spec/unit/helper_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'spec_helper'
+require 'middleman-hashicorp/hashicorp'
+
+class Middleman::HashiCorpExtension
+  describe 'page_source helper' do
+    app = middleman_app
+    instance = Middleman::HashiCorpExtension.new(app, bintray_enabled: false)
+    instance.app = app # unclear why this needs to be set after, but it must
+
+    it 'should return path/filename.extension for non-index files' do
```

Comment:
It's generally considered bad style to start specs with "should": http://betterspecs.org